packages/components/stories/SidePanel.js

```diff
@@ -164,7 +192,7 @@ stories
 				]);
 				const panel = (
 					<SidePanel
-						actions={panelItems}
+						actions={_.shuffle([...items, ...other, ...other, ...other])}
```

Comment:
import shuffle instead of using lodash for some tree shaking reasons

Reply:
I finally removed the use of shuffle.